guides/V1_get_moments_in_conversation.md

```diff
@@ -9,8 +9,10 @@ is: 'beta'
 
       x-graphql-view: BETA
 
+or with <a href="https://developer.watsonwork.ibm.com/tools/graphql?apiType=beta" target="_blank" >apiType=beta on the graphical GraphQL tool URL</a>.
 
-This is a sample GraphQL query to get moments in a space. Be sure to switch "space-id" to the actual id you're curious about.
+
+This is a sample GraphQL query to get moments in a space. Be sure to switch "space-id" to the id of the space you're curious about.
```

Comment:
Minor point, but `spaceId` and `conversationId` in the model are two different things.  The fact that the `id` of the (first) `Conversation` happens to match the `id` of the `Space` is a byproduct of decisions made....

Reply:
Good point. I will update this to getMomentsInSpace since the API calls the parameter spaceID right now I think.